src/components/views/ProductCard.ts

```diff
@@ -25,10 +28,40 @@ export default class ProductCard implements ProductCardInterface {
         this.activePreviewImage = null;
     }
 
+    private createBreadCrumps() {
+        const container = createElem(HTMLTags.DIV, ClassNames.BREADCRUMPS_CONTAINER, '');
+        const stepsLine = createElem(HTMLTags.DIV, ClassNames.STEPS_LINE, '');
+        const linksContainer = createElem(HTMLTags.DIV, ClassNames.BREADCRUMPS_LINKS_CONTAINER, '');
+        const initialLink = createElem(HTMLTags.LINK, '', 'Catalogue') as HTMLLinkElement;
```

Comment:
Maybe instead string 'Catalogue' is ,better to use string-constant. 
In the string-constants.ts we have constant fot this called 'STORE_VIEW_TITLE'

Reply:
Done